### PR TITLE
Adds remove-unused-vars via putout 

### PR DESCRIPTION
**Pull request by danieldelcore**
Exposes remove-unused-vars via putout. 

[https://github.com/coderaiser/putout/tree/master/packages/plugin-remove-unused-variables](https://github.com/coderaiser/putout/tree/master/packages/plugin-remove-unused-variables#readme)

🐊 

CC @coderaiser

### CODE REVIEW

File: community/javascript/remove-unused-vars/transform.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import putout from 'putout';
+// @ts-expect-error
+import removeUnusedVars from '@putout/plugin-remove-unused-variables';
+
+export default function transformer(file: FileInfo) {
+  const output = putout(file.source, {
+    plugins: [removeUnusedVars],
+  });
+
+  return output.code;
```
Comment (danieldelcore) [inline community/javascript/remove-unused-vars/transform.ts:12]: @coderaiser You'll probably be able to spot the issue instantly, but I keep running into this:

image

File: community/javascript/remove-unused-vars/transform.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+}
```
Comment (danieldelcore) [inline community/javascript/remove-unused-vars/transform.ts:13]: @coderaiser You'll probably be able to spot the issue instantly, but I keep running into this:

image

Comment (danieldelcore) [reply community/javascript/remove-unused-vars/transform.ts:13]: Do you know of a way to preserve formatting via recast? 
image

Comment (coderaiser) [reply community/javascript/remove-unused-vars/transform.ts:13]: This is how recast works when it has a little code to work with 🤷. I usually fix this with [`eslint-plugin-putout`](https://github.com/coderaiser/putout/tree/master/packages/eslint-plugin-putout#readme) which solves all formatting issues.

Anyways I’m working on [`@putout/printer`](https://github.com/putoutjs/printer) which just makes good formatting each time, not trying to preserve anything (and much easier in maintenance).

**Pull request merged**